### PR TITLE
Add redirect to home when clicking Technica logo

### PR DESCRIPTION
**Pull request by anna-feng**
Added ability to redirect users that click on Technica logo to the home page

Test by running the platform and clicking on the Technica logo from any page.

### CODE REVIEW

File: frontend/.huskyrc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
   "hooks": {
-    "pre-commit": "vue-cli-service lint --fix"
+    "pre-commit": "./node_modules/.bin/vue-cli-service lint --fix"
```
Comment (burbelohugo) [inline frontend/.huskyrc:3]: Can you remove all changes besides the navbar edit please? So the package-lock.json, package.json, and husky edits should all be removed. You can do this with `git checkout origin/master <FILE_NAME>` in the terminal :)

Comment (burbelohugo) [reply frontend/.huskyrc:3]: Sorry, we should have done this before we wrapped up

Comment (anna-feng) [reply frontend/.huskyrc]: No worries! Did I remove things correctly?

Comment (burbelohugo) [reply frontend/.huskyrc]: Yep!

**Pull request merged**